concent_api/concent_api/urls.py

```diff
@@ -13,14 +13,21 @@
     1. Import the include() function: from django.conf.urls import url, include
     2. Add a URL to urlpatterns:  url(r'^blog/', include('blog.urls'))
 """
-from django.conf.urls import url, include
-from django.contrib   import admin
 
-import core.urls
-import gatekeeper.urls
+from django.conf            import settings
+from django.core.exceptions import ImproperlyConfigured
 
-urlpatterns = [
-    url(r'^',        include(gatekeeper.urls, namespace = 'gatekeeper')),
-    url(r'^api/v1/', include(core.urls, namespace = 'core')),
-    url(r'^admin/',  admin.site.urls),
-]
+from concent_api.constants  import AVAILABLE_CONCENT_FEATURES
+
+
+if not hasattr(settings, 'CONCENT_FEATURES'):
+    raise ImproperlyConfigured("CONCENT_FEATURES variable must be defined in settings")
+
+if not set(settings.CONCENT_FEATURES) - set(AVAILABLE_CONCENT_FEATURES.keys()) == set():
+    raise ImproperlyConfigured("Unrecognized features in CONCENT_FEATURES")
+
+urlpatterns = []
```

Comment:
@rwrzesien 
I'm getting an error from `mypy` here:

```
concent_api/concent_api/urls.py:13: error: Need type annotation for variable)
```

We should probably add a type annotation here, like `urlpatterns: List[...] = []`. But what would be the type of items on the list? How would you handle this?

Reply:
@cameel I would do `# type: ignore` here as this is django stuff.

Reply:
Well, OK for now but I think that we should eventually come up with a better solution. I hope we won't have to have these ignores everywhere.